scene/2d/line_2d.cpp

```diff
@@ -359,6 +387,7 @@ void Line2D::_bind_methods() {
 
 	ADD_PROPERTY(PropertyInfo(Variant::POOL_VECTOR2_ARRAY, "points"), "set_points", "get_points");
 	ADD_PROPERTY(PropertyInfo(Variant::REAL, "width"), "set_width", "get_width");
+	ADD_PROPERTY(PropertyInfo(Variant::OBJECT, "width curve", PROPERTY_HINT_RESOURCE_TYPE, "Curve"), "set_curve", "get_curve");
```

Comment:
Did you test it? I don't think property names with a space are supported.

Reply:
Sorry, I forgot I have to write an underscore. Otherwise, Godot worked fine with this space. I tested without any problem on Windows.